src/styles/main.scss

```diff
@@ -1,124 +1,45 @@
 
 @import "header";
 @import "star";
+@import "card";
 
 body {
   margin: 0;
 }
 
-.product__button-item,
-.product__rate,
-.product__button,
-.product__price {
-  display: flex;
-}
+// Grid:
 
-.card {
-  display: flex;
-  flex-flow: wrap;
-  width: 944px;
-  height: 856px;
+.main {
   margin: 50px auto;
-}
-
-.product {
-  display: inline-block;
-  box-sizing: border-box;
-  font-family: "Roboto", sans-serif;
-  padding: 0;
-  border: 1px solid #f3f3f3;
-  background-color: #fff;
-}
-
-.product--retreat {
-  margin: 0 48px 53px 0;
-  height: 400px;
-  width: 200px;
-}
-
-.product__img {
-  width: 160px;
-  height: 134px;
-  margin: 30px 19px;
-}
-
-.product__text {
-  line-height: 15px;
-  font-weight: 500;
-  margin-top: 11px;
-  margin-left: 14px;
-  font-size: 12px;
-}
-
-.product__code {
-  font-size: 11px;
-  line-height: 13px;
-  margin: 5px 14px;
-  font-weight: 400;
-}
-
-.product__price {
-  margin-top: 0;
-  margin-left: -1px;
-  justify-content: space-between;
-}
-
-.product__info {
-  font-size: 11px;
-  line-height: 13px;
-  margin: 25px 15px;
-}
-
-.product__price-item {
-  font-weight: 700;
-  font-size: 16px;
-  line-height: 18px;
-  margin-top: 21px;
-  margin-right: 14px;
-}
-
-.product__button {
+  display: grid;
   justify-content: center;
-  align-items: center;
-  margin-top: -4px;
-}
-
-.product__rate {
-  margin-left: -1px;
-  justify-content: space-between;
-  margin-top: 15px;
 }
 
-.product__button-item {
-  justify-content: center;
-  align-items: center;
-  line-height: 14px;
-  box-sizing: border-box;
-  font-weight: 700;
-  text-decoration: none;
-  font-size: 14px;
-  text-transform: uppercase;
-  border: 1px solid #00acdc;
-  background: #00acdc;
-  border-radius: 5px;
-  width: 170px;
-  height: 40px;
-  color: #fff;
-}
-
-.product__button-item:hover {
-  background: #fff;
-  color: #00acdc;
-}
-
-.product__respond {
-  font-size: 11px;
-  line-height: 13px;
-  margin-top: 1px;
-  margin-right: 14px;
-}
-
-.card > :nth-child(4),
-.card > :nth-child(8) {
-  margin-right: 0;
+.card {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-gap: 48px;
+  grid-row-gap: 53px;
+}
+
+@media screen and (max-width: 900px) {
```

Comment:
Your navigation bar for narrow screens is not complete, but at least it's usable.